src/plugins/controls/public/control_types/range_slider/range_slider_embeddable.tsx

```diff
@@ -264,8 +264,10 @@ export class RangeSliderEmbeddable extends Embeddable<RangeSliderEmbeddableInput
 
   private buildFilter = async () => {
     const { value: [selectedMin, selectedMax] = ['', ''], ignoreParentSettings } = this.getInput();
+    const availableMin = this.componentState.min;
```

Comment:
nit: destructuring is a little easier to read!

```suggestion
    const { min: availableMin, max: availableMax } = this.componentState;
```